gadgets/trace_dns/gadget.yaml

```diff
@@ -3,97 +3,93 @@ description: trace dns requests and responses
 homepageURL: https://inspektor-gadget.io/
 documentationURL: https://inspektor-gadget.io/docs/latest/
 sourceURL: https://github.com/inspektor-gadget/inspektor-gadget
+datasources:
+  dns:
+    fields:
+      timestamp_raw:
+        annotations:
+          columns.hidden:true
+      timestamp:
+        annotations:
+          template: timestamp
+      src:
+        annotations:
+          template: l4endpoint
+      dst:
+        annotations:
+          template: l4endpoint
+      pid:
+        annotations:
+          template: pid
+      task:
+        annotations:
+          template: comm
+      uid:
+        annotations:
+          template: uid
+          columns.hidden: true
+      gid:
+        annotations:
+          template: uid
+          columns.hidden: true
+      name:
+        annotations:
+          columns.width: 30
+      qr:
+        annotations:
+          columns.width: 2
+      pkt_type:
+        annotations:
+          columns.width: 8
+      rcode:
+        annotations:
+          columns.width: 8
+      latency_ns:
+        annotations:
+          columns.width: 8
+          columns.hidden: true
+      anaddr:
+        annotations:
+          columns.width: 16
+      netns:
+        annotations:
+          template: ns
+      mntns_id:
+        annotations:
+          template: ns
+      tid:
+        annotations:
+          columns.width: 16
+          columns.alignment: left
+          columns.hidden: true
+          columns.ellipsis: end
```

Comment:
These look like the default values - should we just drop them everywhere?

Reply:
I defined a new variable with default annotations and dropped all defaults from all gadget.yaml files.